src/Maestro/Maestro.Web/Api/v2020_02_20/Controllers/BuildsController.cs

```diff
@@ -168,6 +174,17 @@ public override async Task<IActionResult> GetBuildGraph(int id)
             return Ok(new Models.Build(build));
         }
 
+        [HttpGet("{buildId}/commit")]
+        [SwaggerApiResponse(HttpStatusCode.OK, Type = typeof(Build), Description = "The commit matching specified criteria")]
+        [ValidateModelState]
+        public async Task<IActionResult> GetCommit(int buildId)
+        {
+            Data.Models.Build build = await _context.Builds.Include(b => b.Incoherencies).FirstOrDefaultAsync(b => b.Id == buildId);
+            IRemote remote = await _factory.GetRemoteAsync(build.AzureDevOpsRepository ?? build.GitHubRepository, null);
+            Commit commit = await remote.GetCommitAsync(build.AzureDevOpsRepository ?? build.GitHubRepository, build.Commit);
+            return Ok(commit);
```

Comment:
This return value needs to be a new object.  The "Commit" we have is part of the internal operation of IRemote. The API needs to return a version locked representation of that that isn't bound to the internal IRemote implementation in any way.

Like all the other methods that return "Models.Whatever", this should `return Ok(new Models.Commit(commit))`